helm/alfresco-content-services/values.yaml

```diff
@@ -63,18 +63,22 @@ global:
     # is solr6 or SEARCH_USERNAME and SEARCH_PASSWORD keys.
     existingSecretName: null
   auditIndex:
-    # -- Elasticsearch URL
+    # -- set this URL if you have an external search service for audit indexing
     url: null
-    # -- Elasticsearch username
+    # -- set usernname for authentication against the external elasticsearch service for audit indexing
     username: null
-    # -- Elasticsearch password
+    # -- set password for authentication against the external elasticsearch service for audit indexing
     password: null
     # -- Name of the secret managed by this chart
     secretName: &aas_elasticsearch_secretName alfresco-aas-elasticsearch-secret
     # -- Name of an existing secret that contains AUDIT_ELASTICSEARCH_USERNAME and AUDIT_ELASTICSEARCH_PASSWORD keys.
     existingSecretName: null
-    # -- Base url for kibana environment variable `SERVER_PUBLICBASEURL` 
-    publicBaseUrl: null
+  elasticsearch:
+    service:
+      name: elasticsearch
+      ports:
+        restAPI: 9200
```

Comment:
This being part of the global namespace it should not be needed and be available IF people DO enable the chart.
Can you test it?